twins/redis/twins.go

```diff
@@ -74,6 +74,11 @@ func (tc *twinCache) IDs(_ context.Context, channel, subtopic string) ([]string,
 	if err != nil {
 		return nil, errors.Wrap(ErrRedisTwinIDs, err)
 	}
+	idsWildcard, err := tc.client.SMembers(attrKey(channel, twins.SubtopicWildcard)).Result()
```

Comment:
`SMembers` could have better naming

Reply:
Please elaborate on this. `SMembers` is redis function.